#name : try do
#group : control structure
# --
try do
    ${1:value}
catch
  _ -> { nil }
         $0
after
   IO.puts "Didn't catch"
end